### DISCUSSION TITLE
Get notifications - if a Workflow is in 'Queued' Or 'In-Progress' state for more time

### DISCUSSION THREAD

**Discussion by abhishekNKamble** [community/community#31314] [Actions]
I'm working on a scenario, where I would like to get notified - If my workflow is in 'Queued' Or 'In-Progress' state for more than given time duration. 
I haven't came across any relevant GitHub documentation or blogs, proposing an solution for above said scenario.
Any pointers in regards would help.

**Comment by jfullstackdev** [comment]
I don't know whether this update at the Profile Settings will include that

https://github.com/settings/notifications

![image](https://user-images.githubusercontent.com/47092464/187473340-9943b9a7-09a8-4487-8921-a762f52525d9.png)

**Reply by abhishekNKamble** [reply]
I did check this one earlier, it sends notifications only on completion of workflows.
